.devcontainer/post-install.sh

```diff
@@ -1,3 +1,2 @@
 poetry install
 pre-commit install
-pre-commit
```

Comment:
this was not needed :)

tbf we should update the dev container, but can do in another PR